### ISSUE TITLE
Onboarding window is too large and not resizable on macOS

### ISSUE THREAD

**Issue by hra42** [pieces-app/support#463]
### Software

Desktop Application

### Operating System / Platform

macOS

### Your Pieces OS Version

10.1.12

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

The new onboarding creates a window that is too large for the macOS screen. That might be caused by using an external monitor (that currently isn't connected when creating the screenshot). The window can't be resized.

![Image](https://github.com/user-attachments/assets/849badd3-dbe6-4de9-a32d-64cc490104b9)

**Comment by mack-at-pieces** [comment]
@hra42 woah this is crazy. Did this happen on an initial installation on macOS?? Can you share a bit more about the flow you had regarding the external monitor?

**Comment by hra42** [comment]
@mack-at-pieces I got a new macBook Pro 16-inch with M4 Pro and installed the latest macOS version (at the time of the Issue this was 15.1, English Language and Apple Intelligence installed)

I had connected an external 32" 1440p Monitor (2560 x 1440), I disconnected the Monitor (because I needed to do some stuff in our server room and we don't have external monitors there), installed pieces via brew (brew install pieces) and setup the pOS and PfD. When I started PfD the window was too large, I tried restarting the mac any cache or resolution issues would reset. Nothing helped until I connected back to monitor so the window has enough space to show the continue buttons.

What I don't understand is how this is even possible, when looking at the specs, the resolution of the latest macBook Pro is 3456 x2234, so the windows should have enough space to be fully shown.
